etc/errors/anvil.json

```diff
@@ -1058,6 +1145,55 @@
                 ]
               }
             },
+            {
+              "name": "TransactionGasEstimationFailed",
+              "code": 11,
+              "message": "Gas estimation failed: \n{inner}",
+              "fields": [
+                {
+                  "name": "inner",
+                  "type": "GasEstimation"
+                },
+                {
+                  "name": "transaction_data",
+                  "type": "bytes"
+                }
+              ],
+              "doc": {
+                "summary": "Transaction gas estimation failed in anvil-zksync.",
+                "description": [
+                  "This error occurs when a gas estimation for transaction failed.",
+                  "This is a wrapper error that contains a more specific gas estimation error inside it, which provides details about the cause of failure."
```

Comment:
Something about it doesn't feel right; it doesn't feel particularly helpful for users.
Imagine that you're using some tool and see the message "This is a wrapper error that contains a more specific <thing that broke> error inside it". Does that information help you?
At the same time, I can imagine that this information could kinda make sense on the docs website, where you go _purposefully_.

Do we want to use descriptions in `anvil-zksync` output? Who is the target audience of this message?

Reply:
Quite a lot of errors turn to be wrappers over other errors. This warrants a special treatment (https://github.com/matter-labs/zksync-error/issues/39). 

I think a convenient solution would be to introduce the wrapping functionality on zksync-error level and then this line can be added automatically to the specific "Additional information" section, distinct from both "Description" and "Summary" field. 

When https://github.com/matter-labs/zksync-error/issues/39 is resolved, I will go through all anvil errors to flag the wrapped field, and I can remove the "This is a wrapper error..." lines from everywhere.

WDYT?